fed/mosaic-mapping/src/main/java/org/eclipse/mosaic/fed/mapping/ambassador/spawning/VehicleTypeSpawner.java

```diff
@@ -175,11 +176,11 @@ public VehicleType convertTypeAndVaryParameters(RandomNumberGenerator random) {
         );
     }
 
-    Double deviateWithBounds(RandomNumberGenerator random, Double mean, double deviation) {
-        if (mean != null && Math.abs(deviation) > 0.0001) {
+    Double deviateWithBounds(RandomNumberGenerator random, Double mean, Double deviation) {
+        if (mean != null && deviation != null && Math.abs(deviation) > 0.0001) {
```

Comment:
Since we're already importing `MathUtils` we might as well use `MathUtils.EPSILON_D` instead of the magic number

Reply:
I'll use `!MathUtils.isFuzzyZero(deviation)` instead.